web/src/components/Hello.js

```diff
@@ -1,7 +1,18 @@
-import React from 'react';
 import PropTypes from 'prop-types';
+import React from 'react';
+import { Measure, Subhead } from 'rebass';
+
+import Alert from '../styles/Alert';
 
-const Hello = props => <h1>Hello {props.name}!</h1>;
+const Hello = props => (
+  <div>
+    <h1>Hello {props.name}!</h1>
+    <Alert kind="error">
+      <Subhead>Hey-oh!</Subhead>
+      <Measure>This is a fake alert :)</Measure>
+    </Alert>
```

Comment:
I know this is meant to be an example, but my first reaction is along the lines of, "Why do I have to know the component types for the title and message for this alert?"  I.e., would `<Alert kind="error" title="Hey-oh!">This is a fake alert :)</Alert>` make more sense?

Not a blocker for now, just something I wondered about.

Reply:
i think it all depends on how flexible we want our alerts to be, but yes, i think your Alert component props / API makes more sense for the use cases we will likely support